### PR TITLE
OSG 3.6 News section

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/osg-36.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+OSG 3.6 News
```
Comment (brianhlin) [inline docs/release/osg-36.md:1]:
```suggestion
!!! danger "Before considering an upgrade to OSG 3.6&hellip;"
    Due to potentially disruptive changes in protocols, contact your VO(s) to verify that they support token-based
    authentication and/or HTTP-based data transfer before considering an upgrade to OSG 3.6.
    If you do not know which VOs you are currently supporting, contact us at <help@opensciencegrid.org>.

OSG 3.6 News
```

For the time being, we want to pepper these warnings wherever 3.6 is mentioned

File: docs/release/osg-36.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+### 2020-02-26: 3.6 Released
+
+This is the start of a new release series where we introduce major changes.
+One of the major changes is the shift to token based authentication.
+Here is a short list of the differences:
+
+-   GridFTP, GSI, and Hadoop are no longer available
+-   Added packages to support token-based authentication
+-   HTCondor 8.9.11 (HTCondor 8.9.12 release was delayed)
+-   XRootD 5.0.0 (XRootD 5.1.0 release was delayed)
```
Comment (brianhlin) [inline docs/release/osg-36.md:25]:
```suggestion
```

We're not releasing XRootD today

File: docs/release/osg-36.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+-   HTCondor 5.0.0 (HTCondor-CE 6.0.0 release was delayed)
```
Comment (brianhlin) [inline docs/release/osg-36.md:26]:
```suggestion
-   HTCondor 5.0.0
```

I wouldn't say delayed, we just decided to change the versioning scheme

File: docs/release/osg-36.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+-   Gratia probe
+    -   consolidated all batch system probes into the HTCondor-CE probe
+    -   dropped all storage probes
```
Comment (brianhlin) [inline docs/release/osg-36.md:29]:
```suggestion
-   [Gratia Probe 2.0.0](https://github.com/opensciencegrid/gratia-probe/releases/tag/v2.0.0-2): replace all batch system probes with the non-root HTCondor-CE probe
```

File: docs/release/osg-36.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (brianhlin) [inline docs/release/osg-36.md:30]: We should add a line item for osg-configure here

**Review by brianhlin — COMMENTED** [review]
First pass at the OSG 3.6 announcement doc. For the line items with specific software versions, we should link to upstream release notes and add a few sub-bullets highlighting features that are interesting to OSG site admins, esp compared to the 3.5 release versions

**Comment by brianhlin** [comment]
Also, I think your commit from #791 snuck in

File: docs/release/osg-36.md (hunk 5 above)
Comment (brianhlin) [reply docs/release/osg-36.md:33]: @matyasselmeci any high-level things that you want to highlight here? It doesn't have to be as thorough as the upgrade notes

Comment (matyasselmeci) [reply docs/release/osg-36.md:33]: - deprecated RSV config
- dropped unused configuration modules and attributes
- reorganized some configuration (see migration instructions)

**Review by brianhlin — COMMENTED** [review]
These notes look quite spartan and I think could use more fleshing out, similar to what we did with the 3.5.0 release: package update/removal sections, 3.5 EOL admonition, and an admonition explaining the lack of GlideinWMS and XRootD.

I think additional entries to this page for other major changes don't have to be quite as detailed but for the initial release (and for the XRootD release), I think we should be very descriptive

File: docs/release/osg-36.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+This is the start of a new release series where we introduce major changes.
+One of the major changes is the shift to token based authentication.
+Here is a short list of the differences:
+
+-   GridFTP, GSI, and Hadoop are no longer available
+-   Added packages to support token-based authentication
+-   HTCondor 8.9.11 (HTCondor 8.9.12 release was delayed)
+-   HTCondor 5.0.0
+-   [Gratia Probe 2.0.0](https://github.com/opensciencegrid/gratia-probe/releases/tag/v2.0.0-2): replace all batch system probes with the non-root HTCondor-CE probe
+-   [OSG-Configure 4.0.0](https://github.com/opensciencegrid/osg-configure/releases/tag/v4.0.0): upgrade to support OSG 3.6
```
Comment (brianhlin) [inline docs/release/osg-36.md:32]:
```suggestion
-   [OSG-Configure 4.0.0](https://github.com/opensciencegrid/osg-configure/releases/tag/v4.0.0):
    - Deprecated RSV
    - Dropped unused configuration modules and attributes
    - Reorganized some configuration (see [update instructions](updating-to-osg-36.md#osg-configure) for more details)
```

File: docs/release/osg-36.md (hunk 5 above)
Comment (brianhlin) [reply docs/release/osg-36.md:33]: Added suggestion https://github.com/opensciencegrid/docs/pull/790/files#r583835583

File: docs/release/osg-36.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+To update to the OSG 3.6 series, please consult the page on
+[updating between release series](updating-to-osg-36.md).
+
+Latest News
+-----------
+
+### 2020-02-26: 3.6 Released
+
+!!! question "Where are GlideinWMS and XRootD?"
+    XRootD and GlideinWMS are both absent in OSG 3.6.0:
```
Comment (brianhlin) [inline docs/release/osg-36.md:24]:
```suggestion
    XRootD and GlideinWMS are both absent in the initial OSG 3.6 release:
```

File: docs/release/osg-36.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+    regular support will end in **August 2021** and critical bug/security support will end in **February 2022**.
+
+This is the start of a new release series where we introduce major changes.
+One of the major changes is the shift to token based authentication.
+Here is a short list of the differences:
+
+-   GridFTP, GSI, and Hadoop are no longer available
+-   Added packages to support token-based authentication
+-   HTCondor 8.9.11: Initial token support (8.9.12, which will contain default configuration using tokens, was delayed)
+-   HTCondor-CE 5.0.0: Support for tokens and Python 3
```
Comment (brianhlin) [inline docs/release/osg-36.md:40]:
```suggestion
-   [HTCondor 8.9.11](https://htcondor.readthedocs.io/en/latest/version-history/development-release-series-89.html#version-8-9-11): Initial token support (8.9.12, which will contain default configuration using tokens, was delayed)
-   [HTCondor-CE 5.0.0](https://htcondor.github.io/htcondor-ce/releases/#500): Support for tokens and Python 3
```

**Review by brianhlin — APPROVED** [review]
Two suggestions, otherwise LGTM

**Pull request merged**